### PR TITLE
chore: removes kmw prefix use from KMW event docs

### LINKED ISSUE #1062: chore(keymanweb): using 'kmw' as shorthand for 'keyman' is silly
https://help.keyman.com/developer/engine/web/current-version/guide/adding-keyboards

It is shorter, by 3 whole characters. But it is confusing and silly. Let's just use `keyman`!

### PR DESCRIPTION
**Pull request by jahorton**
Fixes #1062.

### CODE REVIEW

File: developer/engine/web/17.0/reference/events/index.md [hunk 1]
```diff
@@ -1,100 +1,110 @@
...
 
 <!-- -->
 
-[`kmw.unloaduserinterface`](kmw.unloaduserinterface)
+[`unloaduserinterface`](keyman/unloaduserinterface)
 :   Called when allow ui clean-up.
 
-<!-- -->
+For example, to define a user function to handle the KeymanWeb core
+kmw.keyboardchange event, include:
```
Comment (mcdurdin) [inline developer/engine/web/17.0/reference/events/index.md:60]:
```suggestion
`keyboardchange` event, include:
```

File: developer/engine/web/17.0/reference/events/index.md [hunk 2]
```diff
@@ -1,100 +1,110 @@
...
 :   Called when allow ui initialization.
 
 <!-- -->
 
-[`kmw.unloaduserinterface`](kmw.unloaduserinterface)
+[`unloaduserinterface`](keyman/unloaduserinterface)
 :   Called when allow ui clean-up.
 
-<!-- -->
+For example, to define a user function to handle the KeymanWeb core
```
Comment (mcdurdin) [inline developer/engine/web/17.0/reference/events/index.md:59]:
```suggestion
For example, to define a user function to handle the KeymanWeb
```

File: developer/engine/web/17.0/reference/events/index.md [hunk 3]
```diff
@@ -1,100 +1,110 @@
...
+kmw.keyboardchange event, include:
 
-[`osk.show`](osk.show)
-:   Called when OSK displayed.
+``` typescript
+keyman.addEventListener('keyboardchange',
+  function(p)
+  {
+    ui.updateMenu(p['internalName'],p['languageCode']);
```
Comment (mcdurdin) [inline developer/engine/web/17.0/reference/events/index.md:66]:
```suggestion
    ui.updateMenu(p.internalName,p.languageCode);
```

Let's fixup some of those unnecessary `['']` google closure compiler things

File: developer/engine/web/17.0/reference/events/index.md [hunk 4]
```diff
@@ -1,100 +1,110 @@
...
 
-<!-- -->
+```
 
-[`osk.hide`](osk.hide)
-:   Called when OSK hidden.
+---
 
-<!-- -->
+### OSK events (on `keyman.osk`)
```
Comment (mcdurdin) [inline developer/engine/web/17.0/reference/events/index.md:73]:
```suggestion
### `keyman.osk` On Screen Keyboard events
```

File: developer/engine/web/17.0/reference/events/index.md [hunk 5]
```diff
@@ -1,100 +1,110 @@
...
-  
-```
+<!-- -->
 
-or a user function to modify the user interface when the on-screen
+[`show`](osk/show)
+:   Called when OSK displayed.
+
+
+For example, to facilitate a user function to modify the user interface when the on-screen
```
Comment (mcdurdin) [inline developer/engine/web/17.0/reference/events/index.md:100]:
```suggestion
For example, add an event handler to modify the user interface when the on-screen
```

File: developer/engine/web/17.0/reference/events/index.md [hunk 6]
```diff
@@ -1,100 +1,110 @@
 ---
-title: Events
+title: Events - main index
```
Comment (mcdurdin) [inline developer/engine/web/17.0/reference/events/index.md:2]:
```suggestion
title: Events - Keyman Engine for Web
```

I don't think we need 'main index' in the title but we should be trying to use the term 'Keyman Engine for Web' consistently

File: developer/engine/web/17.0/reference/events/index.md [hunk 7]
```diff
@@ -1,100 +1,110 @@
...
 ---
-  
+
 A number of KeymanWeb events are exposed to allow the designer of a user
```
Comment (mcdurdin) [inline developer/engine/web/17.0/reference/events/index.md:5]:
```suggestion
A number of events are exposed to allow the designer of a user
```

I don't really like this whole paragraph. There's more to the events than just 'allowing the designer of a user interface to ... ' but probably outside the scope

File: developer/engine/web/17.0/reference/events/index.md [hunk 8]
```diff
@@ -1,100 +1,110 @@
...
+
+For example, to facilitate a user function to modify the user interface when the on-screen
 keyboard is displayed:
 
 ``` typescript
-osk.addEventListener('show',
+keyman.osk.addEventListener('show',
   function(p)
   {
     ui.updateUI(p['x'],p['y'],p['userLocated']);
```
Comment (mcdurdin) [inline developer/engine/web/17.0/reference/events/index.md:107]:
```suggestion
    ui.updateUI(p.x, p.y, p.userLocated);
```

File: developer/engine/web/17.0/reference/events/keyman/index.md [hunk 9]
```diff
@@ -0,0 +1,5 @@
+---
+title: Base-engine events
+---
+
+[Return to main `events` index](..)
```
Comment (mcdurdin) [inline developer/engine/web/17.0/reference/events/keyman/index.md:5]: This should list the keyman events -- copypasta from ../index.md

File: developer/engine/web/17.0/reference/events/osk/index.md [hunk 10]
```diff
@@ -0,0 +1,5 @@
+---
+title: OSK events
+---
+
+[Return to main `events` index](..)
```
Comment (mcdurdin) [inline developer/engine/web/17.0/reference/events/osk/index.md:5]: This should list all the osk events, again copypasta from ../index.md

**Review by mcdurdin — CHANGES_REQUESTED** [review]
This is heading in the right direction.

Some of my comments here relate to some of the wider issues with the documentation, e.g. nomenclature/terminology. I don't think we want to address those across all the documentation in this PR, but we should probably fixup as we go.

**Comment by mcdurdin** [comment]
Oh and this doesn't fix #1062! That was an issue about the use of `kmw` in https://help.keyman.com/developer/engine/web/current-version/guide/adding-keyboards...

File: developer/engine/web/17.0/reference/events/index.md (hunk 5 above)
Comment (jahorton) [reply developer/engine/web/17.0/reference/events/index.md:100]:
```suggestion
For example, to add an event handler that modifies the user interface when the on-screen
```

**Review by mcdurdin — APPROVED** [review]
LGTM

**Pull request merged**